test/paper-input.html

```diff
@@ -196,23 +196,51 @@
         input = fixture('basic');
       });
 
-      test('focus/blur events fired on host element', function() {
-        var nFocusEvents = 0;
-        var nBlurEvents = 0;
+      // At the moment, it is very hard to correctly fire exactly
+      // one focus/blur events on a paper-input. This is because
+      // when a paper-input is focused, it needs to focus
+      // its underlying native input, which will also fire a `blur`
+      // event.
```

Comment:
Previously, this test described a particular behavior on the host: that it fires focus events when it is focused, and blur events when it is blurred. Now, the test only asserts the former.

The expectation that it fires a set number of times is no longer practical, but my feedback is that it still makes sense to set the expectation that `blur` will fire.